src/utils/floating-ui.ts

```diff
@@ -380,6 +380,9 @@ export function connectFloatingUI(
 
   disconnectFloatingUI(component, referenceEl, floatingEl);
 
+  // ensure position matches for initial positioning
+  // floatingEl.style.position = component.overlayPositioning;
+
```

Comment:
How is this a fix, there is no change to the component code other than comments?

Reply:
I forgot to uncomment after some local testing. My bad!